beacon-chain/p2p/types/object_mapping.go

```diff
@@ -179,4 +186,42 @@ func InitializeDataMaps() {
 			return &ethpb.AttesterSlashingElectra{}, nil
 		},
 	}
+
+	// Reset our light client optimistic update map.
+	LightClientOptimisticUpdateMap = map[[4]byte]func() (interfaces.LightClientOptimisticUpdate, error){
+		bytesutil.ToBytes4(params.BeaconConfig().AltairForkVersion): func() (interfaces.LightClientOptimisticUpdate, error) {
+			return &lightclientConsensusTypes.OptimisticUpdateAltair{}, nil
```

Comment:
I don't like this too much. We export a type that should not be exported because everything should be done through constructors and the interface. Someone not knowing about the interface may use it. And we export it only for this one use case.

One solution is to add a new constructor in the light client package such as this:
```
func New() *finalityUpdateAltair {
	return &finalityUpdateAltair{}
}
```
The compiler will complain that we return an unexported type but we can suppress the warning and add comments why this function is there. And then instead of `return &lightclientConsensusTypes.OptimisticUpdateAltair{}, nil` we would have `return &lightclientConsensusTypes.New()}, nil`. Changing `New` to something more meaningful, of course.